### ISSUE TITLE
Ability to transfer custom arguments?

### ISSUE THREAD

**Issue by fmigneault** [bird-house/birdhousebuilder.recipe.nginx#11]
Using Twitcher's pywps [wps.cfg](https://github.com/bird-house/twitcher/blob/master/twitcher/wps.cfg), it is possible to specify output directory and corresponding like so: 
```
[server]
url=https://localhost:5000/ows/wps
sethomedir=true
setworkdir=true
outputpath=/data/wps_outputs
outputurl=https://localhost:5000/wpsoutputs
```

To allow nginx to serve the above path as a file server, the following config must be added to [nginx.conf](https://github.com/bird-house/twitcher/blob/master/templates/nginx.conf) after the `location /` section serving the installed Twitcher app.
```
    location /wpsoutputs
    {
        alias /data/wps_outputs;
        autoindex off;
    }
```

My question is, would it be possible (if not already in some other way?) to add custom inputs to `birdhousebuilder.recipe.nginx` so that parameters like `${settings:wps-output-path}` and `${settings:wps-output-dir}` can be passed to avoid hardcoded values in `nginx.conf`?
Such variables would be useful since they need to match in both nginx and wps configs, and both depend on many pre-configured values in buildout.

**Comment by cehbrecht** [comment]
... my next step would be to make an Ansible recipe for the Twitcher installation. So, if this is not too urgent we can make this more flexibel with Ansible. 

Currently I have a playbook for pywps:
https://github.com/bird-house/ansible-wps-playbook

I want to move the pywps part to its own ansible recipe. And then there could be a Twitcher recipe which could also be used by this playbook.

**Comment by cehbrecht** [comment]
See also: https://github.com/bird-house/ansible-wps-playbook/issues/53

**Comment by fmigneault** [comment]
@cehbrecht what is the advantage of ansible over buildout in this case? I'm not too familiar with it.

**Comment by cehbrecht** [comment]
@fmigneault Well, people have heard of Ansible but not about buildout ;) 

I'm already replacing the pywps setup with Ansible. It takes care of Nginx and postgres on the system level, which is the domain of Ansible. Buildout is meant for App deployment ... I could sill keep it and call buildout from Ansible. But in our case there is probalby not much more then a configuration left. The work is done by Ansible + Conda.

**Comment by huard** [comment]
@cehbrecht If I understood correctly, the whole Copernicus project is moving toward ansible for deployment, correct?

**Comment by cehbrecht** [comment]
@huard that is correct. But the "whole Copernicus" is in this case just the WPS service. The ESGF data node installation, which we also use in Copernicus, is done with bash and python scripts.
